### PR TITLE
[April Fools] Issue of skill

### PR DESCRIPTION
**Pull request by RigglePrime**

### CODE REVIEW

**Comment by Sealed101** [comment]
speedmerge

**Review by ZephyrTFA — APPROVED** [review]

**Comment by iLikeMemes** [comment]
Should be base game

**Comment by Mothblocks** [comment]
For the benefit of the maintainers, April Fools PRs will be labeled and closed. Key Holders are still able to test merge PRs that are closed, so this makes no difference to you, the author, assuming you were able to find a Key Holder to merge this PR in the first place. If you need to make updates to this PR, you can simply add to the branch, and when the TM updates it will use those changes, even if they do not show up to the PR.

**Comment by ATH1909** [comment]
should unironically be merged

**Comment by SupernovaXTS** [comment]
maybe reword it a bit but yeah should be merged
